<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BucketWindow_</class>
 <widget class="QMainWindow" name="BucketWindow_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="bill_plainTextEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>431</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="return_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font: 12pt &quot;Times New Roman&quot;;
	background-color: rgb(255,179,71);
    border-radius: 12;
	border: 1px solid gray
}

QPushButton:hover {
    background-color: rgb(255, 163, 67);
}

</string>
    </property>
    <property name="text">
     <string>Вернуться в корзину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buy_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(85, 187, 255);
    border-radius: 12;
	border: 1px solid gray
}

QPushButton:hover {
    background-color: rgb(31, 174, 233);
}

</string>
    </property>
    <property name="text">
     <string>Купить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
